src/filtrering/filtrering-filter/filterform/hendelser-filterform.tsx

```diff
@@ -50,6 +50,8 @@ export function HendelserFilterform({form, filtervalg, endreFiltervalg}: Hendels
         'AVBRUTT_SOKEAVTALE'
     ];
 
+    const andreMuligheter = ['MAL'];
```

Comment:
er dette nødvendig når det kun er brukt ett sted?

Reply:
tror det er tenkt at det da er lett å gjøre den større? 🤷‍♂️

Reply:
da kan vi vel gjøre det dersom vi gjør den større?

Reply:
samme for meg altså, bare en liten note

Reply:
Kan du sjekke igjen..